File: 1013-fibonacci-number/fibonacci-number.py
class Solution:
    def fib(self, n: int) -> int:
        f=0
        s=1
        if(n==0):
            return 0 
        elif(n<=2):
            return 1
        else:
            while n-2!=0:
                temp=f
                f=s
                s=temp+s
                n-=1
            return f+s

File: 1013-fibonacci-number/fibonacci-number.java
class Solution {
    public int fib(int n) {
        int f=0,s=1;
        if(n==0) return 0;
        if(n<=2) return 1;
        else{
            while(n-3 >= 0){
                int temp = f;
                f=s;
                s=temp+s;
                n--;
            }
            return f+s;
        }
    }
}